TombLib/TombLib/Catalogs/TEN Node Catalogs/Lara.lua

```diff
@@ -6,7 +6,7 @@
 -- !Arguments "Numerical, 20, Air value, [ 0 | 1800 ]"
 
 LevelFuncs.Engine.Node.TestLaraAir = function(operator, value)
-    return LevelFuncs.Engine.Node.CompareValue(TEN.Objects.Lara:GetAir(), value, operator)
+	return LevelFuncs.Engine.Node.CompareValue(TEN.Objects.Lara:GetAir(), value, operator)
```

Comment:
why are there intends like these everywhere? Is it github being github?

Reply:
VS Code automatically formats all rows by doing indentation

Reply:
Can you rollback all formatting and keep it the same as before? Apart from that, I have no objections.